app/views/papers/_sort_by.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="btn-group btn-group-xs nav-button-group sort" role="group" aria-label="...">
+  <%= link_to 'Votes', url_for(sort: 'votes'), class: 'btn btn-default ' + ('active' if params[:sort] != 'pub_date').to_s, role: "button" %>
+  <%= link_to 'Publish Date', url_for(sort: 'pub_date'), class: 'btn btn-default '  + ('active' if params[:sort] == 'pub_date').to_s, role: 'button'  %>
```

Comment:
The wording in the code should represent your intention. So if you mean to set a default, then make it obvious that it's the default sort order. There should be several ways to do this.